### PR TITLE
Merge master into dev

### PR DESCRIPTION
**Pull request by AdamBJohnsonx**
Please make sure every Pull Request has the following information:

* A reference to a related issue in your repository (mandatory)
* A description of the changes proposed in the pull request
* @mentions of the person required for reviewing proposed changes (optional)

### CODE REVIEW

File: userappwithbroker/build.gradle [hunk 1]
```diff
@@ -70,7 +80,7 @@ dependencies {
     implementation "com.google.android.material:material:$rootProject.ext.materialVersion"
 
     localImplementation(project(':adal'))
-    distImplementation("com.microsoft.aad:adal:3.0.2") {
+    distImplementation("com.microsoft.aad:adal:3.1.0") {
```
Comment (AdamBJohnsonx) [inline userappwithbroker/build.gradle:83]:
```suggestion
    distImplementation("com.microsoft.aad:adal:3.1.2") {
```

Comment (AdamBJohnsonx) [inline userappwithbroker/build.gradle:83]:
```suggestion
    distImplementation("com.microsoft.aad:adal:3.1.2") {
```

**Review by gitaumoses4 — APPROVED** [review]
Looks good.

File: adal/build.gradle [hunk 2]
```diff
@@ -139,10 +130,10 @@ dependencies {
         transitive = false
     }
 
-    snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '2.0.11', changing: true)
+    snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '3.0.6', changing: true)
 
     // 'dist' flavor dependencies
-    distApi("com.microsoft.identity:common:2.0.11") {
+    distApi("com.microsoft.identity:common:3.0.6") {
```
Comment (tanmaymanolkar1) [inline adal/build.gradle:136]: the common would be 3.0.8 as of now.

**Review by tanmaymanolkar1 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**